test/fixtures/output/billing/list.golden

```diff
@@ -0,0 +1,4 @@
+      Start Date      |      End Date       | Granularity |  Line Type  | Product | Resource ID | Resource Display Name | Environment ID | Network Access Type |   Price    | Unit | Original Amount | Discount Amount |   Amount     
+----------------------+---------------------+-------------+-------------+---------+-------------+-----------------------+----------------+---------------------+------------+------+-----------------+-----------------+--------------
+  2023-01-01 00:00:00 | 2023-01-02 00:00:00 | DAILY       | LINE_TYPE_1 | KAFKA   | lkc-123     | kafka_1               | env-123        | INTERNET            | 0.12300000 | GB   |     70.00000000 |     20.00000000 | 50.00000000  
```

Comment:
Can we try to format the amounts for the human (non-serialized) output? There should be a helper function inside the `confluent price list` command.

Reply:
I've intentionally kept the floating points to 8 decimal places since these are aggregated by daily amounts. If we truncate the decimal less then we've found that `sum(amounts)` would not match the month-end invoice due to rounding error.

Reply:
thanks, i've move the the function a separate util file under `pkg/utils` so both commands can use the same. It also helps us remove the `Unit` column.

Reply:
Can you move it under `pkg/billing` instead?